packages/zipkin-transport-http/package.json

```diff
@@ -14,12 +14,13 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "dependencies": {
+    "node-fetch": "^2.6.0",
     "zipkin": "^0.21.0"
   },
   "devDependencies": {
     "body-parser": "^1.15.2",
+    "express": "^4.17.1",
```

Comment:
I am curious about this one, how come tests don't complain?

Reply:
My guess is as it is lerna, expressjs was coming from the global package.json

Reply:
deleted express as it is in global package json